Remove beta config var

hypesystem opened the pull request:
Jeg har fjernet _BETA env var'en fra de secrets der bruges i beta - istedet bruger vi nu den uden _BETA, og derfor kunne jeg fjerne special case i configs.

Based on #336 

Jeg har også ryddet lidt ekstra op jeg faldt over

ITViking commented on shared/config/env/base.js:
```diff
@@ -2,7 +2,7 @@
 var path = require('path');
 var fs = require('fs');
-var _ = require('lodash');
+var _ = require('lodash.invert');
```
Mangler renaming til invert

hypesystem replied:
very good thanks

ITViking commented on shared/config/env/beta.js:
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var _ = require('lodash');
+var _ = require('lodash.merge');
```
rename: _ => merge

hypesystem replied:
perf ty

ITViking commented on shared/config.js:
```diff
@@ -1,7 +1,7 @@
 
 const path = require('path');
-const _ = require('lodash');
+const merge = require('lodash.merge');
```
mangler rename på linje 88: `_.merge(config, clientSideConfig);`

hypesystem replied:
Yes!

hypesystem commented:
Fixed the errors you pointed out, collapsed all the PRs into a single one (for testing) and rebased on current master.

Going to test and then release to beta.
